src/test/groovy/org/prebid/server/functional/model/request/auction/Prebid.groovy

```diff
@@ -36,6 +36,7 @@ class Prebid {
     Sdk sdk
     List<AdServerTargeting> adServerTargeting
     BidderControls bidderControls
+    PrebidModulesConfig modules
```

Comment:
I'll leave a separate remark here. Looks like you're reusing a class between request and DB config. This looks incorrect AFAIK. I don't think that all modules that could have account config will be able to take the *same* in the request.

Reply:
updated